client/me/security-2fa-setup-backup-codes/index.jsx

```diff
@@ -97,7 +97,7 @@ class Security2faSetupBackupCodes extends React.Component {
 				<Security2faProgress step={ 3 } />
 				<p>
 					{ this.props.translate(
-						'Backup codes let you access your account if your phone is ' +
+						'Backup codes let you access your account if your unit is ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **29** times:
`translate( 'Backup codes let you access your account if your phone is lost, stolen, or if you run it through the washing machine and the bag of rice trick doesn't work.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).